install/src/main/java/org/jetbrains/bsp/bazel/install/BloopEnvironmentCreator.kt

```diff
@@ -53,15 +54,14 @@ class BloopEnvironmentCreator(
 
 
     private fun refreshProjectArgs(): Try<List<String>> {
-        val pwd = Paths.get("").toAbsolutePath()
         return launcherArgumentCreator.classpathArgv().map {
             listOfNotNull(
                 launcherArgumentCreator.javaBinaryArgv(),
                 Constants.CLASSPATH_FLAG,
                 it,
                 launcherArgumentCreator.debuggerConnectionArgv(),
                 Constants.BLOOP_BOOTSTRAP_CLASS_NAME,
-                pwd.toString(),
+                launcherArgumentCreator.bazelWorkspaceRootDir(),
```

Comment:
I think this changes the semantics of how this used to work.  Before the change it'd always use the current working directory, after the change, if `bazelWorkspaceRootDir` is unset it'll use `workspaceRootDir` first (if set).

I'm actually +1 to the change still, just wanted to make sure I'm understanding what's changing.  We should update the README as well.

Reply:
yea we want to have the following order:
1. bazel workspace root dir flag (`-w`)
2. workspace root dir (`-d`)
3. pwd

I think that we need such order because the "normal" use case is to install server with `-d` inside bazel workspace, and we'd like to keep this behavior without forcing users to use one more flag with exactly the same path